reflex/components/tags/iter_tag.py

```diff
@@ -23,37 +20,36 @@ class IterTag(Tag):
     # The component render function for each item in the iterable.
     render_fn: Callable
 
-    @staticmethod
-    def get_index_var() -> Var:
+    # The name of the index var.
+    index_var_name: str = "i"
+
+    def get_index_var(self) -> Var:
         """Get the index var for the tag.
 
         Returns:
             The index var.
         """
         return BaseVar(
-            _var_name=INDEX_VAR,
+            _var_name=self.index_var_name,
             _var_type=int,
         )
 
-    @staticmethod
-    def get_index_var_arg() -> Var:
+    def get_index_var_arg(self) -> Var:
         """Get the index var for the tag.
```

Comment:
can this docstring help untangle why we have `get_index_var` and `get_index_var_arg` because i think i get it, but it's still not super clear.